src/applications/my-education-benefits/config/form.js

```diff
@@ -305,8 +309,8 @@ function AdditionalConsiderationTemplate(page, formField) {
 }
 
 function givingUpBenefitSelected(formData) {
-  return ['ACTIVE_DUTY', 'SELECTED_RESERVE'].includes(
-    formData[formFields.viewBenefitSelection][formFields.benefitSelection],
+  return ['Chapter30', 'Chapter1606'].includes(
```

Comment:
Changed names to reflect that these labels referred to the actual chapters to be relinquished. It was hard to figure out with current naming.